packages/components/src/CollapsiblePanel/CollapsiblePanel.scss

```diff
@@ -58,12 +62,13 @@ $tc-collapsible-selected-panel-border-color: #6AA9D1 !default;
 	position: relative;
 
 	&.selected {
-		border-left: 4px solid $tc-collapsible-selected-panel-border-color;
+		border-left: 4px solid $tc-collapsible-selected-panel-color;
 		padding-left: 1px;
 	}
 
 	svg {
 		transform-origin: center;
+		vertical-align: text-bottom;
```

Comment:
should be `baseline`, text-bottom stands for position at the bottom of `y`, `p`, `q` or `j`. It's a few exotic!